src/sentry/interfaces/stacktrace.py

```diff
@@ -153,12 +154,20 @@ def remove_function_outliers(function):
     return _ruby_anon_func.sub('_<anon>', function)
 
 
-def remove_filename_outliers(filename):
+def remove_filename_outliers(filename, platform=None):
```

Comment:
This is the only case where platform currently changes grouping. The comment covers why.